params/config.go

```diff
@@ -40,9 +40,9 @@ var (
 	// CypressChainConfig is the chain parameters to run a node on the cypress main network.
 	CypressChainConfig = &ChainConfig{
 		ChainID:                  big.NewInt(int64(CypressNetworkId)),
-		IstanbulCompatibleBlock:  nil,
-		LondonCompatibleBlock:    nil,
-		EthTxTypeCompatibleBlock: nil,
+		IstanbulCompatibleBlock:  big.NewInt(86816005),
+		LondonCompatibleBlock:    big.NewInt(86816005),
+		EthTxTypeCompatibleBlock: big.NewInt(86816005),
```

Comment:
For my curiosity, is it okay that all hard-fork block numbers are the same?

Reply:
1. During the node operation, the current block number is checked whether is bigger than each hard-fork block number respectively. So, assigning the same block number for many hard-fork doesn't affect to the node operation.
2. The order of hard forks is checked in `CheckConfigForkOrder` when a node start. And, all existing hard-forks are allowing the same blocknumbers for successive hard-forks.

Reply:
Thanks for response :)